e2etest/bgp_test.go

```diff
@@ -86,6 +86,11 @@ var _ = ginkgo.Describe("BGP", func() {
 			BGPPort:  179,
 			Password: "ibgp-test",
 		},
+		cc: container.Config{
```

Comment:
Feels a bit confusing having container.Config inside containerConfig, combining the structs should solve this.